heartbeat/LVM-activate

```diff
@@ -441,7 +441,7 @@ tagging_check()
 	# The volume_list must be initialized to something in order to
 	# guarantee our tag will be filtered on startup
 	if ! lvm dumpconfig activation/volume_list; then
-		ocf_log err  "LVM: Improper setup detected"
+		ocf_log_err  "LVM: Improper setup detected"
```

Comment:
These needs to be updated to `ocf_log err`.

Reply:
Sorry for my carelessness , I have completed the modification

Reply:
No worries.